google/cloud/spanner_dbapi/connection.py

```diff
@@ -86,6 +120,20 @@ def list_tables(self):
         )
 
     def run_sql_in_snapshot(self, sql, params=None, param_types=None):
+        """Runs SQL request in snapshot.
+
+        :type sql: str
+        :param sql: SQL request
+
+        :type params: list
+        :param params: (Optional) List of parameters.
+
+        :type param_types: list
+        :param param_types: (Optional) List of parameters' types.
```

Comment:
These are of `dict` not `list` type, see the [Snapshot.execute_sql()](https://github.com/googleapis/python-spanner/blob/754938386c96814a3546d30d38d874734d1c201c/google/cloud/spanner_v1/snapshot.py#L194) method.